src/frontend/screens/Settings/ProjectSettings/YourTeam/index.tsx

```diff
@@ -0,0 +1,208 @@
+import * as React from 'react';
+import {MessageDescriptor, defineMessages, useIntl} from 'react-intl';
+import {
+  NativeNavigationComponent,
+  ViewStyleProp,
+} from '../../../../sharedTypes';
+import {ScrollView, StyleSheet, View} from 'react-native';
+import {Button} from '../../../../sharedComponents/Button';
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
+import {Text} from '../../../../sharedComponents/Text';
+import {BLACK} from '../../../../lib/styles';
+import {DeviceCard} from '../../../../sharedComponents/DeviceCard';
+import {useProjectMembers} from '../../../../hooks/server/projects';
+import {Loading} from '../../../../sharedComponents/Loading';
+import {COORDINATOR_ROLE_ID} from '@mapeo/core/dist/capabilities';
+import {useDeviceInfo} from '../../../../hooks/server/deviceInfo';
+
+const m = defineMessages({
+  title: {
+    id: 'screens.Setting.ProjectSettings.YourTeam.title',
+    defaultMessage: 'Your Team',
+  },
+  inviteDevice: {
+    id: 'screens.Setting.ProjectSettings.YourTeam.inviteDevice',
+    defaultMessage: 'Invite Device',
+  },
+  coordinators: {
+    id: 'screens.Setting.ProjectSettings.YourTeam.coordinators',
+    defaultMessage: 'Coordinators',
+  },
+  particpants: {
+    id: 'screens.Setting.ProjectSettings.YourTeam.particpants',
+    defaultMessage: 'Particpants',
+  },
+  coordinatorDescription: {
+    id: 'screens.Setting.ProjectSettings.YourTeam.coordinatorDescription',
+    defaultMessage:
+      'Coordinators can invite devices, edit and delete data, and manage project details.',
+  },
+  deviceName: {
+    id: 'screens.Setting.ProjectSettings.YourTeam.deviceName',
+    defaultMessage: 'Device Name',
+  },
+  dateAdded: {
+    id: 'screens.Setting.ProjectSettings.YourTeam.dateAdded',
+    defaultMessage: 'Date Added',
+  },
+  particpantDescription: {
+    id: 'screens.Setting.ProjectSettings.YourTeam.particpantDescription',
+    defaultMessage:
+      'Participants can take and share observations. They cannot manage users or project details.',
+  },
+});
+
+export const YourTeam: NativeNavigationComponent<'YourTeam'> = ({
+  navigation,
+}) => {
+  const {formatMessage: t} = useIntl();
+  const membersQuery = useProjectMembers();
+  const deviceInfo = useDeviceInfo();
+  const coordinators = React.useMemo(
+    () =>
+      !membersQuery.data
+        ? undefined
+        : membersQuery.data.filter(
+            member =>
+              member.capabilities.name === 'Coordinator' ||
+              member.capabilities.name === 'Project Creator',
```

Comment:
is there a better way of determining the role? right now it is not strongly typed : `member.capabilities.name` is just type `string`

Reply:
Yes good point, we need to think this through, I'd like to loop @EvanHahn in and maybe have a quick chat about it with you.

Reply:
Happy to chat about this any time.